lib/python/rose/config.py

```diff
@@ -97,6 +97,7 @@
 import shlex
```

Comment:
I've made a very small number of changes to this file since you took it out of my branch, most of these are to fix doctests. There don't appear to be any conflicts.

Reply:
Sure, I anticipated there might be some changes but was planning to wait until nearer a 'ready' stage to go back to make any adjustments. We should ensure that the most recent version of files you have ported end up in the final branch that gets reviewed for merging into ``master``.